src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFunctionSelect.tsx

```diff
@@ -26,13 +29,15 @@ type Props = {
 
 type State = {
   submitting: boolean;
+  toggleAllState: boolean;
 };
 
 const getLabel = (func: LambdaFunction) => func.FunctionName;
 
 export default class AwsLambdaFunctionSelect extends React.Component<Props, State> {
   state = {
     submitting: false,
+    toggleAllState: true,
```

Comment:
I would prefer for this to be computed from the models values. Can we do that?

Reply:
@EvanPurkhiser We are purposely doing that because of the weird state that happens if you have some enabled but some disabled. What would the value of the switch be then?

Reply:
If you have some enabled but others disabled I would expect the switch to be off.

If you want to turn all off you would then flip it to on and then back to off

Reply:
I feel like that's sort of over-engineering the problem. AJ and I are both OK with the state of the toggle all button not matching the underlying state of the lambdas.

Reply:
IMO it's just a bit confusing as a user to select everything, and then turn it off and have it still on indicating everything is selected.